models/schema.sql

```diff
@@ -1,5 +0,0 @@
-DROP DATABASE IF EXISTS exampledb;
```

Comment:
make sure you add the correct database and table that we will be using from now on.